concepticondata/conceptlists/Nurse-1975-1038.tsv

```diff
@@ -0,0 +1,1038 @@
+958	Nurse-1975-1038-957	exercise; drive out a devil	v.	exercise; drive out a devil//exercise, drive out a devil (-	(ku)punga (pepo)		
+959	Nurse-1975-1038-958	wave, let off a trap, remove a	v.	wave, let off a trap, remove a	(ku)tegua	978	WAVE
+960	Nurse-1975-1038-959	jealousy	v.	sprain the angkle//jealousy	wivu	2016	JEALOUSY
+961	Nurse-1975-1038-960	ostrich	n.	ostrich	mbuni		
+962	Nurse-1975-1038-961	new	n.	new	-pya	1231	NEW
+963	Nurse-1975-1038-962	charm (esp. to ensure wife's f	n.	charm (esp. to ensure wife's f	tego	1271	DELIGHT
+964	Nurse-1975-1038-963	rest the cheek on the hand (in	v.	rest the cheek on the hand (in	(ku)shika tama		
+965	Nurse-1975-1038-964	spring, machine	n.	spring, machine	mtambo	144	MACHINE
+966	Nurse-1975-1038-965	small cucumber	n.	small cucumber	tango		
+967	Nurse-1975-1038-966	slip, be slippery	v.	slip, be slippery	(ku)teleza	1324	SLIP
+968	Nurse-1975-1038-967	shiver, shudder	v.	shiver, shudder	(ku)tapa	52	SHIVER
+969	Nurse-1975-1038-968	put a pot on the fire	v.	put a pot on the fire	(ku)teleka		
+970	Nurse-1975-1038-969	slash	v.	slash	(ku)tema		
+971	Nurse-1975-1038-970	be settled, be in good order	v.	cut firewood//be settled, be in good order	(ku)tengenea		
+972	Nurse-1975-1038-971	be listless	v.	be listless	(ku)tepetea		
+973	Nurse-1975-1038-972	be loose, faint, weak	v.	be loose, faint, weak	(ku)legea	1601	WEAK
+974	Nurse-1975-1038-973	place, put	v.	place, put	(ku)tia	998	PUT
+975	Nurse-1975-1038-974	be left over, remain over	v.	be left over, remain over	(ku)saa, (ku)salia		
+976	Nurse-1975-1038-975	set up, appoint	v.	set up, appoint	(ku)simika		
+977	Nurse-1975-1038-976	long straight hair (of animals	n.	long straight hair (of animals	usinga		
+978	Nurse-1975-1038-977	tingle with excitement	v.	tingle with excitement	(ku)sisimika		
+979	Nurse-1975-1038-978	mud, mire	n.	mud, mire	33//tope	640	MUD
+980	Nurse-1975-1038-979	sink, be drowned	v.	sink, be drowned	(ku)tota	1088	SINK (DESCEND)
+981	Nurse-1975-1038-980	put together, compose	v.	put together, compose	(ku)tunga	1743	JOIN
+982	Nurse-1975-1038-981	strike with a spear	v.	strike with a spear (=fumo)//strike with a spear	(ku)fuma (fumo)		
+983	Nurse-1975-1038-982	want, need, wish	v.	want, need, wish	(ku)taka	1784	WANT
+984	Nurse-1975-1038-983	open	v.	open	(ku)fungua	1156	OPEN
+985	Nurse-1975-1038-984	teach, instruct	v.	teach, instruct	(ku)fundisha, (ku)funza	709	TEACH
+986	Nurse-1975-1038-985	put forth, produce, display	v.	put forth, produce, display	(ku)toa		
+987	Nurse-1975-1038-986	think, imagine	v.	think, imagine	(ku)waza	2271	THINK
+988	Nurse-1975-1038-987	rib	n.	rib	ubavu	801	RIB
+989	Nurse-1975-1038-988	apply by stretching, spread ov	v.	apply by stretching spread ove//apply by stretching, spread ov	(ku)wamba		
+990	Nurse-1975-1038-989	hold, arrest	v.	held, arrest//hold, arrest	(ku)bamba		
+991	Nurse-1975-1038-990	squeeze oneself up against wal	v.	squeeze oneself up against wal//squeeze onself up against a wa	(ku)jibanza		
```

Comment:
But here, you can see, @chrzyki, that my other theory, that we have problems with filling in the slots is correct. So these cases should best be identified.